doc/userguide/rules/meta.rst

```diff
@@ -211,3 +211,29 @@ The format is::
 If the value is src_ip then the source IP in the generated event (src_ip
 field in JSON) is the target of the attack. If target is set to dest_ip
 then the target is the destination IP in the generated event.
+
+requires
```

Comment:
I think it should be mention that this is parsed first, before any other syntax validation is done

Reply:
> When parsing rules, the parser attempts to process the 'requires'
keywords before others. This allows it to occur after keywords that
may only be present in specific versions of Suricata, as specified by
the 'requires' statement. However, the keywords preceding it must
still adhere to the basic known formats of Suricata rules.